packages/calcite-components/src/components/input-text/input-text.scss

```diff
@@ -292,10 +301,12 @@ input[type="text"]::-ms-reveal {
 .prefix {
   @apply order-2;
   border-inline-end-width: theme("borderWidth.0");
+  inline-size: var(--calcite-input-text-prefix-width, auto);
```

Comment:
because input, input-number, and input-text are nearly identical and share most of the same styles/props I think we can simplify this down to `--calcite-input-prefix-size` and `--calcite-input-suffix-size` for all components.